### DISCUSSION TITLE
Prism.DependencyObjectExtensions.CheckAccess unable to find DispatcherQueue property

### DISCUSSION THREAD

**Discussion by sigmarsson** [PrismLibrary/Prism#2523] [Uno Platform]
Please take a quick look at [this issue](https://github.com/microsoft/microsoft-ui-xaml/issues/5601) to judge if it is a `WinUI / Uno` error or any API has changed, which is not yet harmonized. Thanks. It had worked prior to an update was made to `<PackageReference Include="Microsoft.ProjectReunion" Version="0.8.1" />`

**Comment by sigmarsson** [comment]
WinUI API has changed.

"In Reunion 0.8 DispatcherQueue moved from the Microsoft.System to the Microsoft.UI.Dispatching namespace."

**Reply by sigmarsson** [reply]
@brianlagunas , will you update Prism WinUI to harmonize again with the latest `ProjectReunion 0.8.1` ? WinUI moved the `DispatcherQueue` in a different name space. ... or only with 1.0 ?

**Reply by brianlagunas** [reply]
WinUI is not a priority for us right now. Feel free to submit a PR if you have the time.

**Reply by sigmarsson** [reply]
@brianlagunas , okay I tried and sent you one : https://github.com/PrismLibrary/Prism/pull/2530

**Reply by sigmarsson** [reply]
@brianlagunas , CI bogged down; `The type or namespace name 'Dispatching' does not exist in the namespace 'Microsoft.UI' (are you missing an assembly reference?)`
shall we update the CI as well in order to the 0.8.2 ProjectReUnion assemblies be taken ?

**Reply by brianlagunas** [reply]
yeah, that will need to be done. @dansiegel is the owner of all things CI. I'll let him handle that when he recovers from his surgery.

**Reply by sigmarsson** [reply]
Merci.

**Reply by sigmarsson** [reply]
@brianlagunas , maybe could you please roll out a `pre nuget` inclusive this PR ?

**Reply by sigmarsson** [reply]
@brianlagunas , could you please tell how to forge a nuget for myself out of my dev branch ?

**Reply by dansiegel** [reply]
any time we merge to master a package is published to Sponsor Connect. This is only available to those who are active GitHub sponsors of either @brianlagunas or myself. Note that I'll take a look at the CI issue probably next week as I'm still recovering from surgery.

**Comment by crogeniks** [comment]
When can we expect this fix to publicly available? With WinUI dropping 1.0 I want to explore prism and compare to other frameworks, but this issue is blocking my exploration of Prism.

**Comment by sigmarsson** [comment]
Yeah it is strange that no1 of the WinUI community intends to incorporate Prism into their WinUI application. For (almost) a year ?

**Reply by brianlagunas** [reply]
Seems no one really cares enough about WinUI and Prism to make the effort. The community has spoken.

**Reply by crogeniks** [reply]
I don't know about spoken. The root cause was fixes by a member of the community. The PR was approved and merged back in august. In December, no public NuGet version with the fix was released.

I tried, although yes not very hard, no just compile Prism and see if it fits my needs. I didn't and waited for a NuGet update. 

It has now been 10 months with no update on public NuGet update. I personally just moved on and decided to try another framework.

I totally understand that the maintainers of Prism are doing this as a side thing. I do not nor will never expect quick updates for free. I do believe the Sponsor connect is a good thing for the project. But at a certain point, when the general community is left out of all updates.. 🤷‍♂️

**Reply by edoust** [reply]
I used a custom build of Prism compiled for Uno/WinUI for a while, but then removed it, since IMO it doesn't offer any benefit anymore in the Uno/WinUI scenario. I now use what `CommunityToolkit` offers and further trimmed down the app so it runs cross-platform incl. with Blazor

Could you point out which parts of Prism you would like to use? Just curious how others see this issue

**Reply by sigmarsson** [reply]
@edoust i.e. `: RegionAdapterBase<T>`

**Comment by sigmarsson** [comment]
Hi Brian. blunt but still find it spooky because should we aggregate the **WinUI+Uno** communites, there must be more individuals than just me. 
Perhaps the majority deserted because many controls related issues have not been stabilized for a long time ? for exapmple - `ContentDialog`. My project bogged down because `XamDataChart` is still in alpha, `XamDockManager` isn't even visible on the horizon, Prism isn't compiling due to this one. So IG invests primarily in low-code solutions and/or Angular, I do not know but hardly believe that commercial grade products are competitive without those controls. (for which I personaly would be paying)

**Reply by brianlagunas** [reply]
There is no mass adoption of WinUI by the market. There is no driving force, commercial or otherwise, that would encourage us to focus on, or even care about WinUI right now. The majority of the current market is web (Blazor, Angular, React, etc).

**Reply by sigmarsson** [reply]
Uno team, will [make up](https://github.com/unoplatform/uno/discussions/8405#discussioncomment-2486001) the control's repertoire and furnish the Chart control for WinUI desktop. Would it be possible the community obtains a patched Prism for Uno like 8.1.98, just containing this patch, nothing else?
